backend-implicaction/src/test/java/com/dynonuggets/refonteimplicaction/controller/UserControllerTest.java

```diff
@@ -0,0 +1,320 @@
+                .andExpect(jsonPath("$.contribution", Matchers.is(userDto.getContribution())))
+                .andExpect(jsonPath("$.registeredAt", Matchers.is(userDto.getRegisteredAt().toString())))
+                .andExpect(jsonPath("$.activatedAt", Matchers.is(userDto.getActivatedAt().toString())))
+                .andReturn();
+    }
+
+    @Test
+    void getUserByIdWithoutJwtShouldBeForbidden() throws Exception {
+
+        UserDto userDto = UserDto.builder()
+                .id(2L)
+                .username("paul-sdv")
+                .firstname("Paul")
+                .lastname("Flu")
+                .email("paul@implicaction.fr")
+                .url("www.google.fr")
+                .hobbies("surf,gaming,judo")
+                .purpose("")
+                .registeredAt(Instant.now())
+                .activatedAt(Instant.now())
+                .roles(roles)
+                .active(true)
+                .build();
+
+        when(userService.getUserById(userDto.getId())).thenReturn(userDto);
+        mvc.perform(MockMvcRequestBuilders.get("/api/users/" + userDto.getId())
+                .accept(MediaType.ALL).contentType(MediaType.APPLICATION_JSON)).andDo(MockMvcResultHandlers.print())
+                .andExpect(MockMvcResultMatchers.status().isForbidden())
+                .andReturn();
+    }
+
+    @Test
+    @WithMockUser
+    void getAllFriendsForOneUserShouldReturnUserList() throws Exception {
+
+        int first = 0;
+        int rows = 10;
+
+        UserDto sender = UserDto.builder()
+                .id(2L)
+                .username("paul-sdv")
+                .firstname("Paul")
+                .lastname("Flu")
+                .email("paul@implicaction.fr")
+                .url("www.google.fr")
+                .hobbies("surf,gaming,judo")
+                .purpose("")
+                .registeredAt(Instant.now())
+                .activatedAt(Instant.now())
+                .roles(roles)
+                .active(true)
+                .build();
+
+        UserDto receiver = UserDto.builder()
+                .id(3L)
+                .username("paul-sdv")
+                .firstname("Paul")
+                .lastname("Flu")
+                .email("paul@implicaction.fr")
+                .url("www.google.fr")
+                .hobbies("surf,gaming,judo")
+                .purpose("")
+                .registeredAt(Instant.now())
+                .activatedAt(Instant.now())
+                .roles(roles)
+                .active(true)
+                .build();
+
+
+        ArrayList<UserDto> friendsList = new ArrayList<>();
+        friendsList.add(receiver);
+
+        Page<UserDto> userPageMockResponse = new PageImpl<>(friendsList);
+        Pageable pageable = PageRequest.of(first, rows);
+        ResultActions actions;
+
+        when(relationService.getAllFriendsByUserId(sender.getId(), pageable)).thenReturn(userPageMockResponse);
```

Comment:
```suggestion
        when(relationService.getAllFriendsByUserId(anyLong(), any()).thenReturn(userPageMockResponse);
```
Tu devrais pouvoir simplifier comme ça et donc supprimer ton pageable.